.github/workflows/integration-test.yml

```diff
@@ -21,8 +21,8 @@ jobs:
           node-version: 14.17.6
```

Comment:
not part of this , but we can upgrade node to 16 LTS

Reply:
The package's baseline is 14. I.e. it means it runs on 14+. So we should keep this as is until we move baseline up.